src/components/BrowseNavbar.tsx

```diff
@@ -0,0 +1,156 @@
+import { useEffect, useRef, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { signOut } from "firebase/auth";
+import { Bell, ChevronDown, Search } from "lucide-react";
+import logo from "../assets/logo.png";
+import { auth } from "../utils/firebase";
+
+const BrowseNavbar = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const dropdownRef = useRef<HTMLDivElement>(null);
+  const navigate = useNavigate();
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  // Close the dropdown when clicking outside
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
+      ) {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, []);
+
+  function handleSignOut() {
+    signOut(auth)
+      .then(() => {
+        navigate("/login");
+      })
+      .catch((error) => {
+        console.log("error=", error);
+      });
+  }
+  return (
+    <div className="absolute w-screen z-50 text-white bg-gradient-to-b from-black">
+      <div className="flex justify-between">
+        <div className="flex">
+          <div>
+            <img
+              src={logo}
+              alt="logo"
+              height={100}
+              width={190}
+              className="ml-10"
+            />
+          </div>
+          <ul className="flex space-x-3 mt-6 ml-6 cursor-pointer">
+            <li>
+              <p>Home</p>
+            </li>
+            <li>
+              <p>TV Shows</p>
+            </li>
+            <li>
+              <p>Movies</p>
+            </li>
+            <li>
+              <p>New & Popular</p>
+            </li>
+            <li>
+              <p>My List</p>
+            </li>
+            <li>
+              <p>Browse by Languages</p>
+            </li>
+          </ul>
+        </div>
+        <div className="mt-6 mr-10">
+          <ul className="flex space-x-4 cursor-pointer">
```

Comment:
same goes here, we can use the component to just pass the array value and it will provide the ordered list, eliminating code duplication view and follow Clean Code  principles

Reply:
resolved